source/ReleaseNotes/index.rst

```diff
@@ -0,0 +1,10 @@
+.. _rel_notes:
+
+==================================================
+Percona Server for MongoDB |version| Release Notes
+==================================================
+
+.. toctree::
+   :maxdepth: 1
+
+   1.0.0
```

Comment:
should we also add 0.3.0, 0.2.0 and 0.1.0 releases?

Reply:
Peter S said that we should start with 1.0.0